docker/Dockerfile

```diff
@@ -117,9 +117,11 @@ COPY --chown=notify:notify . .
 
 ENV SQLALCHEMY_DATABASE_URI=postgresql://postgres:postgres@localhost/
 
-RUN pip3 install -r requirements_for_test.txt
+RUN pip3 install -r requirements.txt
```

Comment:
we shouldn't need this line (as we copy the prod requirements across on line 112, which includes utils etc)

Reply:
It needs both to:
- install app requirements (including utils)
- use utils to copy `requirements_for_test_common.txt`
- install test requirements including those in `requirements_for_test_common.txt`